[AI-3856] Add FIPS Verified Agent Integrations doc page (#27839)
DataDog/documentation: content/en/integrations/guide/fips-integrations.md

---
title: FIPS Verified Agent Integrations
further_reading:
- link: "/agent/configuration/fips-compliance"
  tag: "Documentation"

>   text: "Datadog FIPS Compliance"

Reviewer: ```suggestion
  text: "Datadog FIPS Compliance"
- link: "agent/guide/fips-agent"
  tag: "Documentation"
  text: "Datadog FIPS Agent"
```

algolia:
  rank: 80
  tags: ["fips", "compliance", "fedramp", "govcloud"]
---

> 

Reviewer: Does this apply to this doc?

```suggestion
{{< callout url="" btn_hidden="true" header="Try the FIPS Agent Preview!">}}
A FIPS flavor of the Datadog Agent is in Preview. For more information, see <a href="/agent/guide/fips-agent">FIPS Agent</a>.
{{< /callout >}}

{{< site-region region="us,us3,us5,eu,ap1" >}}
<div class="alert alert-warning">The Datadog Agent FIPS Proxy is available only in the US1-FED region.</a></div>
{{< /site-region >}}
```

> {{< callout btn_hidden="true" header="Who is this relevant to?">}}
> Customers that require FIPS compliant services and use Datadog integrations.
> {{< /callout >}}
> 

Reviewer: ```suggestion
## Overview
```

> As part of the FedRAMP High effort, a number of integrations have been verified for **FIPS 140-2** compliance. Integrations that are not mentioned below may function in compliance with FIPS 140-2 but have not been tested internally.

Reviewer: ```suggestion
As part of the FedRAMP High effort, a number of integrations have been verified for **FIPS 140-2** compliance. Integrations that are not mentioned below may function in compliance with FIPS 140-2 but have not been tested internally. 

This guide is for customers that require FIPS compliant services and use Datadog integrations.
```

> ## Enabling FIPS Mode for a Supported Integration

Reviewer: ```suggestion
## Enabling FIPS mode for a supported integration
```

To ensure compliance, make sure to use an HTTPS endpoint whenever possible and follow the integration-specific instructions below.

> Integrations marked with "OOTB" require no further configuration.

Reviewer 2: ```suggestion
Integrations marked out of the box ("OOTB") require no further configuration.
```

| Azure Active Directory  | OOTB                                                                                                                                                  |
| CoreDNS                 | OOTB                                                                                                                                                  |
| Elasticsearch           | OOTB                                                                                                                                                  |
| Envoy                   | OOTB                                                                                                                                                  |
| Haproxy                 | OOTB                                                                                                                                                  |
| Istio                   | OOTB                                                                                                                                                  |

> | Kafka                   | To enable TLS make sure to follow this guide.                                                                                                         |

Reviewer: Which guide?

> | MongoDB                 | The `tls` option must be set to true through the integration config.                                                                                  |

Reviewer: ```suggestion
| MongoDB                 | The `tls` option must be set to `true` through the integration configuration.                                                                                  |
```

> | MySQL                   | The `ssl` option must be set through the integration config.                                                                                          |

Reviewer: Set? 

```suggestion
| MySQL                   | The `ssl` option must be set through the integration configuration.                                                                                          |
```

| Nginx                   | OOTB                                                                                                                                                  |
| Php-fpm                 | Even though the `php_fpm` integration uses the random module, that use is restricted to randomizing the retry delay.                                  |
| Postfix                 | OOTB                                                                                                                                                  |
| RabbitMQ                | OOTB                                                                                                                                                  |

> | Redis                   | The `ssl` option must be enabled through the integration config.                                                                                      |

Reviewer: ```suggestion
| Redis                   | The `ssl` option must be enabled through the integration configuration.                                                                                      |
```

| SSH                     | OOTB                                                                                                                                                  |
| TLS                     | OOTB                                                                                                                                                  |

> | Tomcat                  | To enable TLS make sure to follow this guide.                                                                                                         |

Reviewer: Which guide?

| Vault                   | OOTB                                                                                                                                                  |

> | vSphere                 | Both `ssl_verify` and `rest_api_options > tls_verify` need to be set to true if using the vSphere REST API to get tags (`collect_tags: true`).        |

Reviewer: ```suggestion
| vSphere                 | Both `ssl_verify` and `rest_api_options > tls_verify` need to be set to `true` if using the vSphere REST API to get tags (`collect_tags: true`).        |
```

| Windows Service         | OOTB                                                                                                                                                  |

> | Zookeeper               | The `use_tls` option must be enabled through the integration config.                                                                                  |

Reviewer: ```suggestion
| Zookeeper               | The `use_tls` option must be enabled through the integration configuration.                                                                                  |
```

<div class="alert alert-warning">

> **IIS integration:** Configuring this integration to query remote systems is discouraged. It uses a Windows API over which we have no control as far as cryptography goes.

Reviewer: ```suggestion
Configuring the <strong>IIS integration<strong> to query remote systems is discouraged. It relies on a Windows API for cryptography, which Datadog cannot control.
```

</div>

> ## Need Support?
> 
> Contact **Agent Integrations** for assistance.

Reviewer: Can you clarify, how customers should reach out to Agent Integrations? 

```suggestion
## Further reading

{{< partial name="whats-next/whats-next.html" >}}

```

Author: I think that was just wrong, I'm removing it as you suggested.
Customers should contact Datadog support in case of a configuration issue.
